app/mailers/notifier.rb

```diff
@@ -50,8 +50,9 @@ def order_receipt(args)
 
   def review_orders(args)
     @user = User.find(args[:user_id])
-    @accounts = Account.find(args[:account_ids]).map(&:account_list_item).to_sentence
-    send_nucore_mail @user.email, text("views.notifier.review_orders.subject")
+    @accounts = Account.find(args[:account_ids]).map { |a| "#{a.account_list_item}, #{a.owner.user.full_name}" }.join("\n\n")
```

Comment:
Part of me feels that the string rendering should happen in the template. The Notifier is kind of like a controller.